### PR TITLE
#3076 Annotations added to all .mo except package.mo, .mo split

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes [#3076 ](url)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -141,121 +73,37 @@ dynawo/3rdParty/libxml/sax/parser/include/xml/sax/parser/ParserVersion.h
...
+*.blg
+*.dvi
+*.gz
+*.log
 *.orig
+*.out
+*.pdf
+*.pyc
+*.toc
+*.zip
```
Comment (joyelfeghali) [inline .gitignore:93]: The modifications done here are too generic it might ignore files we need, it's better to leave it as it is.

Comment (ErwanGuichard49) [reply .gitignore:93]: When a .tex file is compiled, several additional files are generated : .aux, .log, .out, .pdf, .synctex.gz and (if there is a table of contents) .toc.

.log and .pdf can be found elsewhere but .aux, .out, .synctex.gz and .toc are specific to a .tex compilation.

Moreover, the .bak-mo files are generated when a .mo is deleted through OpenModelica, they ought not to be saved.

File: dynawo/sources/Common/Dictionaries/validateDictionaries.py [hunk 2]
```diff
@@ -332,14 +333,15 @@ def generate_modelica_files(self, modelica_package, modelica_dir=None):
...
 *
-* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools
-* for power systems.
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
 */
 ''')
-            mo_file.write("within " + modelica_package + ";\n\n")
-            mo_file.write('encapsulated package ' + name + 'Keys\n\n')
+            mo_file.write('\npackage ' + name + 'Keys\n')
```
Comment (joyelfeghali) [inline dynawo/sources/Common/Dictionaries/validateDictionaries.py:340]: we can keep encapsulated

File: .gitignore
Comment (joyelfeghali) [inline .gitignore:1]: maybe we can do it in a separate PR

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: dynawo/sources/Common/Dictionaries/validateDictionaries.py (hunk 2 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Common/Dictionaries/validateDictionaries.py]: encapsulated restored

**Review by joyelfeghali — APPROVED** [review]
